soap/src/java/com/zimbra/soap/admin/message/QuarantineDeviceRequest.java

```diff
@@ -73,8 +79,12 @@ public AccountSelector getAccount() {
         return this.account;
     }
 
+    public Boolean getUserDeviceOnly() { return useDeviceOnly; }
+
+    public void setUserDeviceOnly(Boolean userDeviceOnly) { this.useDeviceOnly = userDeviceOnly; }
```

Comment:
Inconsistency in names for attrs `userDeviceOnly` & `useDeviceOnly` , should be same.
if we change the name as suggested above could follow here

Reply:
done